### PR TITLE
Release Godot 4.3 dev 4 and publish release notes

### PR DESCRIPTION
**Pull request by clayjohn**
Ready for review and posting!

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-3-dev-4.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+### AnimationMixer feature parity and other animation goodies
+
+AnimationMixer continues to receive several fixes and enhancements after being introduced in 4.2 to bring it up to parity with AnimationPlayer.
+
+- [GH-86629](https://github.com/godotengine/godot/pull/86629) adds a `CallbackModeDiscrete` option to AnimationMixer to significantly improve the behaviour when blending continuous and discrete tracks.
+
+- [GH-86661](https://github.com/godotengine/godot/pull/86661) introduces several fixes to how audio is handled by AnimationPlayers.
+
+- [GH-86715](https://github.com/godotengine/godot/pull/86715) expands the AnimationMixer and AnimationPlayer APIs with `AnimationMixer::capture()` and `AnimationPlayer::play_with_capture()` which can substitute the old capture update mode. The capture update mode has been difficult to use and prone to issues for awhile now. The introduction of AnimationMixer highlighted many of those issues. The new `capture()` and `play_with_capture()` functions allow you to do the same things as before in a way that is much more aligned with Godot's API and should work much better in general.
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-dev-4.md]:
```suggestion
- [GH-86715](https://github.com/godotengine/godot/pull/86715) expands the AnimationMixer and AnimationPlayer APIs with `AnimationMixer::capture()` and `AnimationPlayer::play_with_capture()` which can substitute the old capture update mode. The capture update mode has been difficult to use and prone to issues for a while now. The introduction of AnimationMixer highlighted many of those issues. The new `capture()` and `play_with_capture()` functions allow you to do the same things as before in a way that is much more aligned with Godot's API and should work much better in general.
```

File: collections/_article/dev-snapshot-godot-4-3-dev-4.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+This snapshot comes loaded with new features and important fixes, after three weeks of further development.
+
+As a reminder, this section only covers changes made since the previous [4.3 dev 3 snapshot](/article/dev-snapshot-godot-4-3-dev-3/). For a more comprehensive overview of what's new in Godot 4.3 compared to 4.2, you'll have to wait for the first beta release, or refer to our [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.3).
+
+### AnimationMixer feature parity and other animation goodies
+
+AnimationMixer continues to receive several fixes and enhancements after being introduced in 4.2 to bring it up to parity with AnimationPlayer.
+
+- [GH-86629](https://github.com/godotengine/godot/pull/86629) adds a `CallbackModeDiscrete` option to AnimationMixer to significantly improve the behaviour when blending continuous and discrete tracks.
```
Comment (akien-mga) [inline collections/_article/dev-snapshot-godot-4-3-dev-4.md]:
```suggestion
- [GH-86629](https://github.com/godotengine/godot/pull/86629) adds a `CallbackModeDiscrete` option to AnimationMixer to significantly improve the behavior when blending continuous and discrete tracks.
```

File: collections/_article/dev-snapshot-godot-4-3-dev-4.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+
+Additionally, [GH-87250](https://github.com/godotengine/godot/pull/87250) adds support for selecting, copying, pasting, and duplicating keyframes within the AnimationPlayer.
+
+### .NET code quality and usability improvements
+
+C# received several fixes and usability improvements in this development release. Among them are [GH-87890](https://github.com/godotengine/godot/pull/87890) which improves the handling of C# generic types. With this base scripts are shown in the "Add Node" and "Create Resource" dialogs even when they are generic types. Between [GH-88469](https://github.com/godotengine/godot/pull/88469) and [GH-88570](https://github.com/godotengine/godot/pull/88570) there has been a lot of cleanup of diagnostic rules and warnings resulting in a much smoother experience. 
+
+As a temporary workaround, we have disabled the ability to generate a signal callback in C# from the editor ([GH-87952](https://github.com/godotengine/godot/pull/87952)). This feature has long been broken for C# and has led to a lof of confusion. We will re-enable this once we have it working reliably (including for external editors).
```
Comment (akien-mga) [inline collections/_article/dev-snapshot-godot-4-3-dev-4.md]:
```suggestion
As a temporary workaround, we have disabled the ability to generate a signal callback in C# from the editor ([GH-87952](https://github.com/godotengine/godot/pull/87952)). This feature has long been broken for C# and has led to a lod of confusion. We will re-enable this once we have it working reliably (including for external editors).
```

**Review by akien-mga — APPROVED** [review]
Looks great! 👍

File: collections/_article/dev-snapshot-godot-4-3-dev-4.md (hunk 2 above)
Comment (clayjohn) [reply collections/_article/dev-snapshot-godot-4-3-dev-4.md]: I can't keep my Canadian voice out of my writing it seems

File: collections/_article/dev-snapshot-godot-4-3-dev-4.md (hunk 3 above)
Comment (clayjohn) [reply collections/_article/dev-snapshot-godot-4-3-dev-4.md]:
```suggestion
As a temporary workaround, we have disabled the ability to generate a signal callback in C# from the editor ([GH-87952](https://github.com/godotengine/godot/pull/87952)). This feature has long been broken for C# and has led to a lot of confusion. We will re-enable this once we have it working reliably (including for external editors).
```

**Comment by clayjohn** [comment]
Thank you for the reviews!

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**